blueocean-dashboard/pom.xml

```diff
@@ -12,4 +12,16 @@
     <packaging>hpi</packaging>
 
     <url>https://wiki.jenkins-ci.org/display/JENKINS/Blue+Ocean+Plugin</url>
+    <build>
```

Comment:
@i386 Reverted. Although in blueocean-dashboard, FB reported bug related to generated Java file for Messages files, I fixed that. That message property is not even used, just had to lower the case to keep FB happy.